onnx_tf/backend.py

```diff
@@ -585,12 +577,22 @@ def _pool(cls, node, input_dict, pool_func, can_pad_zero):
 
     # By default, do not pad
     pad = "VALID"
-
-    if "pads" in node.attrs.keys():
+    pads = node.attrs["pads"] if "pads" in node.attrs else None
+
+    # Pooling1D
```

Comment:
Why do we have to special case 1-dimensional pooling?
As per TF documentation here (https://www.tensorflow.org/api_docs/python/tf/nn/pool), pooling op does support 1d pooling. 

Quote from the documentation:
> For N=1, the valid values are "NWC" (default) and "NCW".

Reply:
`1D` means `NWC` or `NCW`

Reply:
@tjingrant 
For reference, 
https://github.com/tensorflow/tensorflow/blob/7c324811f10eaf049735ace7b6a228ccc931d8ff/tensorflow/python/layers/pooling.py#L32